vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md

```diff
@@ -1,136 +1,101 @@
-- Click *Save*.
+## Edit Images
+You can flip, rotate and crop newly uploaded images.
 
-To dynamically crop an image:
-- Edit the asset by opening *Actions* menu > Edit.
-  - Buttons representing your pre-defined crop ratios are below the image. Clicking any of those buttons changes the selected crop area based on the chosen ratio.
-  - You can create multiple crops of a single image as part of the upload process. The right side of the screen displays the list of crops created during the current session.
+1. Go to `Content` → `Assets` → `Add` 
+2. After uploading an image, click on the actions button to the right of the image. From the drop-down menu, Select Edit.
+![Add Digital Assets Window](./img/assets-edit-image.png)
+3. Use the arrows to flip, rotate or recenter your image. When done, click the `Crop` check button. Multiple resized images can be created at this time. Click `Free` to remove resizing ratio restrictions.
+4. Click `Done`.
+5. Click Add to save all versions.
 
-![image](./img/cms_digital_assets_crop.png)
+## Embed an Asset on a Page
+The uploaded asset can be embedded on a page by using `<@wp.resourceUrl>your-path/your-asset.jpg`. The path can be found in the App Builder at `Administration` → `File Browser`. (e.g. 
+`<img src="<@wp.resourceUrl> cms/images/butterfly.jpg">`
 
-- Use your mouse or the buttons below the image to move or change the crop area, then click *Save* to crop the selected area. Repeat the process as many times as needed.
-  - Click *Close* to return to the Digital Assets screen.
+![File Browser](./img/assets-filebrowser.png)
 
-## Embedding an Asset in a Page
-The uploaded asset can be embedded in a page by using `<@wp.resourceUrl path/to/asset`. The path can be found in the CMS Configuration > File Browser page.
+## Include an Asset in a Content
+The asset can be included in a Content Type that has an `Attach` or `Image` attributes. Follow the [Content Type tutorial](./content-types-tutorial.md) for detailed steps or follow the briefly outlined steps here.
 
-![image](./img/cms_file_browser.png)
+Create a Content Type with an image attribute. 
+1. From the left sidebar select `Content` → `Types` → `Add`
 
-Example usage:
-`<img src="<@wp.resourceUrl cms/images/butterfly_d0_d3.jpg"`
+![Create Content Type](./img/assets-add-type.png)
 
-## Including an Asset in a Content
-The asset can be included in a content of a type that has an *Attach* or *Image* attribute.
+2. Fill out `Code`, `Name` and click `Add`.  
+3. This will take you to the Add Attribute page. In this example, we are using the Attribute Type of image. Enter the `Code` and `Name` as required.  All other fields are optional. Click `Continue` at bottom of page. Note the Attribute Code field is used below to reference the image in the Template.
+![image](./img/assets-add-attribute.png)
+4. Next create the Template or Model where the content type is added. Go to `Content` → `Template` → `Add`
```

Comment:
can omit "Next"

content type -> Content Type

create a template/model where the type is added reads like you create these in the neighborhood of where you see an add option for Content Type